server.rb

```diff
@@ -119,7 +119,7 @@ def admin_get_message
     redirect to ("/")
   end
 
-  post "/playlist" do
+  get "/playlist" do
```

Comment:
Why did this change? This is creating something in the database each time it's called; it shouldn't be a `GET` request.